src/convrnx.c

```diff
@@ -1322,6 +1322,12 @@ static int convrnx_s(int sess, int format, rnxopt_t *opt, const char *file,
     }
     str->time=str->tstart;
     
+    /*Reset the stream*/
+    if (!(str=gen_strfile(format,opt->rcvopt))) {
```

Comment:
See also https://github.com/rtklibexplorer/RTKLIB/pull/454 Might have tried simply reopening the file but it might not have  been right as there is data gathered in the first pass that wants to be retained. Perhaps separate this patch from the unicore patch.